src/test/resources/org/rust/ide/formatter/fixtures/auto_indent/chain_call4_after.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    let moo = foo().bar().baz()
+                   .moo()
+                   <caret>;
```

Comment:
I'm not sure whether it's desired behaviour. Having complete statement (already placed `;`) we may give us a hint, that user might want to continue chain call.